compiler/passes/errorHandling.cpp

```diff
@@ -797,6 +903,21 @@ bool isCheckErrorStmt(Expr* e)
   return false;
 }
 
+Symbol* getErrorSymbolFromCheckErrorStmt(Expr* e)
```

Comment:
I can't seem to find where this function is called, or if it used to be in another part of the compiler. Could you point it out to me?

Reply:
It's not called right now but I'd need it if I try again to make progress on non-inlined iterators that throw. I could remove it from this PR if you like.

Reply:
You can leave it in, but it might be good to add a comment.